core/src/test/java/org/apache/iceberg/util/TestTableScanUtil.java

```diff
@@ -125,8 +123,7 @@ public void testTaskGroupPlanning() {
 
     CloseableIterable<ScanTaskGroup<ParentTask>> taskGroups =
         TableScanUtil.planTaskGroups(CloseableIterable.withNoopClose(tasks), 128, 10, 4);
-
-    Assert.assertEquals("Must have 3 task groups", 3, Iterables.size(taskGroups));
+    Assertions.assertThat(taskGroups).as("Must have 3 task groups").hasSize(3);
```

Comment:
why `Assertions.assertThat` here when you're using `assertThat` in all the other places in this file?